Added Documentation about Incompatible Extensions (#35)
Exafunction/codeium-docs: windsurf/getting-started.mdx

  <Frame>
        <img src="/assets/import-vscode.png"/>
  </Frame>

## Incompatible Extensions

> There are a few extensions that are incompatible with Windsurf. These include other AI code complete extensions and proprietary extensions. You can not install extensions through any marketplace on Windsurf.

Reviewer: ```suggestion
There are a few extensions that are incompatible with Windsurf. These include other AI code complete extensions and proprietary extensions. You cannot install extensions through any marketplace on Windsurf.
```
